python/knowhere/knowhere.i

```diff
@@ -145,7 +145,7 @@ class IndexWrap {
  public:
     IndexWrap(const std::string& name, const int32_t& version) {
         GILReleaser rel;
-        if (knowhere::UseDiskLoad(name, version)) {
+        if (name == std::string("DISKANN")) {
```

Comment:
a magic constant?

Reply:
why not use USeDiskLoad api here?

Reply:
> why not use USeDiskLoad api here?

UseDiskLoad will cause hnsw running with filemanager, this will lead ci fail